src/pages/ControllersIndex/ControllersIndex.tsx

```diff
@@ -298,9 +307,21 @@ function Details() {
       </div>
       <ControllersOverview />
       <div className="l-controllers-table u-overflow--auto">
-        {rows.length > 0 && <MainTable headers={headers} rows={rows} />}
+        {rows.length > 0 && (
+          <>
+            <h5 className="u-hide--large">{Label.DEFAULT}</h5>
```

Comment:
The titles for the tables appear in the header for the first column so I've added these titles above the tables on small screens.